Infrastructure/Services/UserService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Infrastructure.Interfaces;
+using Infrastructure.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.DataAccess
+{
+    public class AddressService : IUserService
```

Comment:
We should really think about approach of this one; We will have two kinds of users (registered and non registered). So if that's the case unregistered user could be just a "temporary" entry for the parcel, and it could be duplicated, as same address could have different name/surname/phone number etc. so would be hard to set business key by which we make it unique

Reply:
Hence the parcel would be owner of this address instance, and we would never querry just the address itself